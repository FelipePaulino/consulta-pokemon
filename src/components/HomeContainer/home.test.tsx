import 'jest-styled-components';
import { screen, render } from '@testing-library/react';
import Home from './index';

describe('Home component', () => {
  it('renders Home', () => {
    render(
      <Home />
    );
    const home = screen.getByTestId('home-component')
    expect(home).toBeInTheDocument();
    expect(home).toHaveStyleRule('background', `url('/images/pokemon-hero.jpg') no-repeat center`);
    expect(home).toMatchSnapshot();
  });
});
