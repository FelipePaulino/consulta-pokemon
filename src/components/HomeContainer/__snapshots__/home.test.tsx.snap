// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders Home 1`] = `
.c0 {
  width: 100%;
  height: 81vh;
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
  background: url('/images/pokemon-hero.jpg') no-repeat center;
  background-size: cover;
}

.c1 {
  color: #FFFFFF;
  width: 509px;
  text-align: center;
}

<div
  class="c0"
  data-testid="home-component"
>
  <h1
    class="c1"
  >
    Cuidamos bem do seu pokémon, para ele cuidar bem de você
  </h1>
</div>
`;
